lib/llm/src/http/service/service_v2.rs

```diff
@@ -427,7 +430,37 @@ impl HttpServiceConfigBuilder {
         all_docs.extend(openapi_docs);
 
         // Add span for tracing
-        router = router.layer(TraceLayer::new_for_http().make_span_with(make_request_span));
+        // Add on_response callback for logging response status code
+        router = router.layer(
```

Comment:
Question: is the main thing here to distinguish different types of error? I thought that the default trace layer logs any errors - but may be mistaken